Description: Silence error messages about text not loaded.
 compiz (ring) - Warn: No compatible text plugin loaded
 compiz (core) - Error: Plugin 'text' not loaded.
Author: Miguel A. Colón Vélez <debian.micove@gmail.com>
Forwarded: no
Last-Update: 2012-03-12

--- a/plugins/ring/ring.xml.in
+++ b/plugins/ring/ring.xml.in
@@ -13,6 +13,7 @@
 			</relation>
 			<requirement>
 				<plugin>opengl</plugin>
+				<plugin>text</plugin>
 			</requirement>
 		</deps>
 		<options>
